addons/survey/static/src/js/survey_form.js

```diff
@@ -81,39 +84,62 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     */
     _onBreadcrumbClick: function (event) {
         event.preventDefault();
-        this._submitForm({'previous_page_id': $(event.currentTarget).data('previousPageId')});
+        this._submitForm($(event.currentTarget));
     },
 
     _onSubmit: function (event) {
         event.preventDefault();
-        if ($(event.currentTarget).val() === 'previous') {
-            this._submitForm({'previous_id': $(event.currentTarget).data('previousId')});
-        } else {
-            this._submitForm({});
-        }
+        this._submitForm($(event.currentTarget));
     },
 
-    _submitForm: function (params) {
+    _submitForm: function ($target) {
         var self = this;
-        var $form = this.$('form');
-        var formData = new FormData($form[0]);
+        var params = {};
+
+        if (self.options.isStartScreen) {
+            params = {'start': true};
```

Comment:
Couldn't we find another way to do this?

I'm thinking of maybe calling a specific route in that case.
Using a "form" route to submit a single "start" parameter seems very hacky.